dashboard/db/migrate/20170823193832_create_section_hidden_scripts.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSectionHiddenScripts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :section_hidden_scripts do |section_hidden_scripts|
+      section_hidden_scripts.belongs_to :section, null: false
+      section_hidden_scripts.belongs_to :script, null: false
```

Comment:
Do we want timestamps in this table? Presumably not (if modeled after `SectionHiddenStage`), but seems worth asking.

Is the intent that we add and (hard) delete rows to this column to reflect current state (as with `SectionHiddenStage`)?